api/events/handlers.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const wrap = require('co').wrap
+const Boom = require('boom')
```

Comment:
For saving in all handlers files requiring the module Boom, you can use this module: https://www.npmjs.com/package/hapi-boom-decorators
Using this you have a more elegant way to return exceptions like: 
```javascript
reply.notFound(message);
```